utils.js

```diff
@@ -65,3 +65,15 @@ function debounce(func, delay) {
 async function addDelay(milliSeconds) {
   await new Promise((resolve) => setTimeout(resolve, milliSeconds));
 }
+
+async function signout() {
+  try {
+    await fetch('https://api.realdevsquad.com/auth/signout', {
```

Comment:
Could you please use the constants we have for this URL?

Reply:
ok